openc3/lib/openc3/packets/packet.rb

```diff
@@ -151,12 +155,10 @@ def target_name=(target_name)
           if !(String === target_name)
             raise(ArgumentError, "target_name must be a String but is a #{target_name.class}")
           end
-
           @target_name = target_name.upcase.freeze
         else
           @target_name = nil
         end
-        @target_name
```

Comment:
I understand this is redundant, but I like explicit returns.   Should be "return @target_name"

Reply:
A setter should not have a return. That's why I removed it ... not because it was implicit

Reply:
https://ruby-doc.org/3.2.2/syntax/methods_rdoc.html#label-Return+Values